.github/ISSUE_TEMPLATE/4-release_checklist.md

```diff
@@ -17,17 +17,18 @@ assignees: ''
 
 **Before release**:
 - [ ] Check [SPEC 0](https://scientific-python.org/specs/spec-0000/) to see if we need to bump the minimum supported versions of GMT, Python and core package dependencies (NumPy/Pandas/Xarray)
-- [ ] Run `make codespell` to check common misspellings. If there are any, either fix them or add them to `ignore-words-list` in `pyproject.toml`
+- [ ] Check repositories in `.pre-commit-config.yaml` and see if there are new versions for the pre-commit hooks
```

Comment:
Actually, we can run `pre-commit autoupdate` to update the pre-commit hooks

Reply:
Can remove this entry after #3414 and #3415 since the config will be auto-updated by pre-commit.ci

Reply:
Already removed in https://github.com/GenericMappingTools/pygmt/pull/3374/commits/aff71d2fb406a9cd0ac33026d19d69bb129fb236